openmdao/core/group.py

```diff
@@ -2598,102 +2597,31 @@ def _get_auto_ivc_out_val(self, tgts, remote_vars, all_abs2meta, abs2meta):
                     meta = abs2meta[tgt]
                     val = meta['value']
                     if has_src_inds:
-                        src_inds, sz = self._get_src_inds_max(tgt, meta)
-                        loc_ranges.append((tgt, sz, src_inds, val, False))
+                        src_idx_found.append(tgt)
                     else:
                         info.append((tgt, meta['size'], val, False))
                 else:
                     info.append((tgt, 0, np.zeros(0), True))
+
             elif dist:  # distributed and local everywhere
-                if has_src_inds:
-                    if tgt in abs2meta:
-                        src_indices = abs2meta[tgt]['src_indices']
-                        rnginfo = (np.min(src_indices), np.max(src_indices), len(src_indices))
-                        dist_ranges.append((tgt, self.comm.allgather(rnginfo)))
-                    else:
-                        # no part of distrib var is local, e.g., a distrib comp under a
-                        # parallel group
-                        dist_ranges.append((tgt, self.comm.allgather((0, 0, 0))))
-                else:  # assume default distrib layout
-                    if tgt in abs2meta:
-                        info.append((tgt, abs2meta[tgt]['size'], abs2meta[tgt]['value'], True))
-                    else:
-                        info.append((tgt, 0, np.zeros(0), True))
+                # OpenMDAO currently can't create an automatic IndepVarComp for inputs on
+                # distributed components.
+                msg = 'Distributed component input "{}" cannot be connected to an automatic '
+                msg += 'IndepVarComp.  Please add one manually and connect it.'
```

Comment:
I think that the whole "AutoIVC thing is supposed to be transparent to the user?
May be better to just say that distributed components require an IndepVarComp?